master/docs/manual/cfg-workers-docker.rst

```diff
@@ -216,6 +216,10 @@ In addition to the arguments available for any :ref:`Latent-Workers`, :class:`Do
     Address of the master the worker should connect to. Use if you master machine does not have proper fqdn.
     This value is passed to the docker image via environment variable ``BUILDMASTER``
 
+``hostconfig``
+    (optional)
+    Extra host configuration parameters passed as a dictionary used to create HostConfig object. See `docker-py's HostConfig documentation <http://docker-py.readthedocs.org/en/latest/hostconfig/>`_ for all the supported options.
+
```

Comment:
Please add short notice about this feature into release notes: https://github.com/buildbot/buildbot/blob/master/master/docs/relnotes/index.rst

Also I would appreciate if you include notice about your previous PR #2029 into release notes.